packages/airnode-node/src/constants.ts

```diff
@@ -50,3 +50,6 @@ export const BASE_FEE_MULTIPLIER = 2;
 
 // The maximum character length of an error message sent on-chain
 export const MAXIMUM_ONCHAIN_ERROR_LENGTH = 100;
+
+// The default gas limit
+export const GAS_LIMIT = 500_000;
```

Comment:
Nit: It's not clear what this "gas limit" applies to. I would prefer a more descriptive name here if possible. 

```suggestion
export const DEFAULT_FULFILLMENT_GAS_LIMIT = 500_000;
```
or maybe `API_CALL_FULFILLMENT_GAS_LIMIT` (?)

Also, the comment above doesn't really add much in it's current state